misc/horizontal-scale-file-create.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -e
+
+N="$1"
+if [ -z "$N" ]; then
+    N=1
+fi
+
+function do_scale_test {
+    T=$(mkdir /cephfs-$N)
```

Comment:
This one confuses me too. The path should be in `/cephfs`?

I suggest creating a directory under root for this test (so it can fragment; root dir cannot fragment). Something like:

````
TESTDIR="/cephfs/$(basename "$0")/"
mkdir -p "$TESTDIR"
T=$(mktemp -d -p "$TESTDIR")
````